index.html

```diff
@@ -0,0 +1,294 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./dist/css/style.css" />
+    <link rel="stylesheet" href="./dist/css/custom.css" />
+    <title>Grid Framework</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <nav class="row d-flex flex-wrap pt-30 pb-30">
+
+                <div class="nav-left-wrapper xs-3 sm-3 md-3 lg-3 xl-3" id="nav-left-wrapper">
+                        <a href="" class="submit_group header_link">
+                            <span class="icon icon-submit pr-10"></span>
+                            <span class="submit-text small-hide d-flex-sm d-flex-md d-flex-lg d-flex-xl">Submit Store</span>
+                        </a>
+                </div>
+
+                <div class="nav-center-wrapper xs-6 sm-6 md-6 lg-6 xl-6" id="nav-center-wrappe">
+                        <a href="" class="header_link">
+                            <span class="icon icon-logo"></span>
+                        </a>
+                </div>
+
+                <div class="nav-right-wrapper xs-3 sm-3 md-3 lg-3 xl-3" id="nav-right-wrapper">
+                        <a href="" class="login_group header_link">
+                            <span class="icon icon-account pr-10"></span>
+                            <span class="small-hide d-flex-sm d-flex-md d-flex-lg d-flex-xl ">Login</span>
+                        </a>
+                </div>
+            </nav>          
+        <div class="hero_content ta-c">
+            <h3 class="row">Curating the Best of Shopify.</h3>
+            <p class="row">We collect and highlight the most beautiful commerce experiences on Shopify to help inspire designers, developers and merchants <span></span>to create amazing things.</p>
+            <div class="buttons_wrapper">
+                    <a class="btn popup-trigger" href="#submit-popup" data-effect="mfp-zoom-in">Submit a Store</a>
+                    <a class="btn start-a-shopify-store" href="https://www.shopify.com/?ref=commerce-cream" target="_blank"><span class="icon icon-shopify-bag"><span class="path1"></span><span class="path2"></span></span><span>Start a Shopify Store</span></a>
+                    <a class="btn" href="/learn-more">Learn More</a>
+            </div>
+        </div>
+        
+        </header>
+        <div class="page_content">
```

Comment:
**Optional:** Use the `<main>` tag here to complete your usage of Semantic HTML on your page.